owa/app/js/components/view_log_entries/viewLogEntries.jsx

```diff
@@ -9,127 +9,284 @@
+                                            placeholderText="From Date"
+                                            maxDate={moment()}
+                                            showMonthDropdown
+                                            showYearDropdown
+                                            dropdownMode="select"/>
+                                        <span>-</span>
+                                        <DatePicker
+                                            className="form-control"
+                                            selected={this.state.endDate}
+                                            selectsEnd
+                                            startDate={this.state.startDate}
+                                            endDate={this.state.endDate}
+                                            onChange={this.handleChangeEnd}
+                                            placeholderText="To Date"
+                                            maxDate={moment()}
+                                            showMonthDropdown
+                                            showYearDropdown
+                                            dropdownMode="select"/>
+                                    </div>
+                                    <div className="col-sm-6 col-md-4">
+                                        <label className="search_lbl" name="gen_by">Generated By</label>
+                                        <Input value={this.state.generatedByValue} name="generatedBy" onChange={this.handleSearch}/>
+                                    </div>
+                                    <div className="col-sm-6 col-md-4">
+                                        <label className="search_lbl">Comment Contains</label>
+                                        <Input id="comment" name="comment" value={this.state.searchFilters.comment} onChange={this.handleSearch}/>
+                                    </div>
+                                    <div className="clear"></div>
+                                </fieldset>
+                                <div className="btn-group" role="group">
+                                    <input
+                                        type="button"
+                                        value="Search"
+                                        className=" col-sm-6 button confirm"
+                                        onClick={this.advancedSearch}/>
+                                        <input type="button" className="col-sm-6 cancelbtn" value="Reset" onClick={this.resetSearchForm}/>
+
+                                </div>
+                            </form>
+                        </div>}</div>
                     <div className="table-responsive logs_table">
+                        { (this.state.searchFilters.source || this.state.searchFilters.identifier || this.state.generatedByValue || this.state.searchFilters.comment || this.state.startDate|| this.state.dateChanged)  &&
```

Comment:
Line too long

Reply:
Your whole file seems to have more than 2spaces of indentation, look into that.

Reply:
Was using 4 spaces probably because that's what I used to do for Python projects but just searched online & I've noticed most JS developers are pro soft indentation so I've reduced them to 2.